src/lib/components/activity/activity-options-update-drawer.svelte

```diff
@@ -0,0 +1,304 @@
+<script lang="ts">
+  import type { WorkflowExecution } from '@temporalio/client';
+
+  import Button from '$lib/holocene/button.svelte';
+  import Checkbox from '$lib/holocene/checkbox.svelte';
+  import DrawerContent from '$lib/holocene/drawer-content.svelte';
+  import Drawer from '$lib/holocene/drawer.svelte';
+  import Input from '$lib/holocene/input/input.svelte';
+  import NumberInput from '$lib/holocene/input/number-input.svelte';
+  import Label from '$lib/holocene/label.svelte';
+  import { translate } from '$lib/i18n/translate';
+  import { updateActivityOptions } from '$lib/services/workflow-activities-service';
+  import { toaster } from '$lib/stores/toaster';
+  import { refresh } from '$lib/stores/workflow-run';
+  import type { ActivityOptions } from '$lib/types';
+  import type { PendingActivity } from '$lib/types/events';
+  import { formatSecondsAbbreviated } from '$lib/utilities/format-time';
+
+  type Props = {
+    open: boolean;
+    namespace: string;
+    execution: WorkflowExecution;
+    activity: PendingActivity;
+  };
+
+  const fromDurationToNumber = (duration) => {
```

Comment:
Nit: probably want to type the `duration` parameter.

Reply:
Fixed